recipes/cwinpy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "cwinpy" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3caedf3aac6b8ef9bc0df8467e1610587c1fae3b2a0266ee6c839d45a79e5cb0
+
+build:
+  entry_points:
+    - cwinpy_knope = cwinpy.knope.knope:knope_cli
+    - cwinpy_knope_dag = cwinpy.knope.knope:knope_dag_cli
+    - cwinpy_knope_generate_pp_plots = cwinpy.knope.testing:generate_pp_plots
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - astropy
+    - bilby >=0.5.8
+    - bilby_pipe >=0.2.0
+    - gwpy
+    - lintegrate
+    - matplotlib
+    - numba
+    - numpy >=1.15
+    - pycondor
+    - python >=3.6
+    - python-lal
+    - python-lalpulsar
+    - scipy
+
+test:
+  imports:
+    - cwinpy
+  commands:
+    - cwinpy_knope --help
+    - cwinpy_knope_dag --help
+    - cwinpy_knope_generate_pp_plots --help
+
+about:
+  home: "https://github.com/cwinpy/cwinpy/"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "CW Inference in Python"
+  doc_url: "https://cwinpy.readthedocs.io/en/latest/"
+  dev_url: "https://github.com/cwinpy/cwinpy/"
+  description: |
+    A Python module for performing Bayesian inference for continuous gravitational-wave signals from pulsars.
+extra:
```

Comment:
Please add empty line between the two sections.

Reply:
Done